src/components/SearchBar.jsx

```diff
@@ -0,0 +1,32 @@
+export function SearchBar({ data, setDisplayData, setSearch, search }) {
+	const handleInputChange = (e) => {
+		const searchTerm = e.target.value;
+		setSearch(searchTerm);
+
+		const filteredUsers = data.filter((item) =>
+			item.name.toLowerCase().includes(searchTerm.toLowerCase()),
+		);
+		setDisplayData(filteredUsers);
+	};
```

Comment:
Adding .trim() to the search term would help prevent users from filtering with spaces.